docs/guides/run/health-checks.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Healthchecks
+description: description
+icon: heart-pulse
+---
+
+## Configuration
+
+Below you can find an example with a toml configuration file for a service with healthcheck enabled.
+
+```toml
+name = 'mysvc'
+.
+.
+.
+[[ports]]
```

Comment:
I'd remove this as this is unrelated. I may also mention in the docs that the port you use for the healthchecks doesn't need to be exposed to the outside world.

Reply:
fixed